.eslintrc

```diff
@@ -14,7 +14,8 @@
     "strict": 0,
     "no-unused-vars": 0,
     "no-undef": 0,
-    "no-console": 0
+    "no-console": 0,
+    "no-underscore-dangle": 0
```

Comment:
This just stops eslint from yelling when you declare an underscored variable like `_foo`